src/main/java/tuteez/model/person/lesson/Lesson.java

```diff
@@ -26,6 +27,27 @@ public class Lesson {
     private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HHmm");
     private static final String VALID_TIME_RANGE_REGEX = "([01]?[0-9]|2[0-3])[0-5][0-9]-([01]?[0-9]|2[0-3])[0-5][0-9]";
     private static final DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HHmm");
+    /**
+     * Maps various string representations of days (e.g., "monday", "mon") to their corresponding Day enum values.
+     * All keys are stored in lowercase.
+     */
+    private static final Map<String, Day> DAY_NAME_MAP = new HashMap<>();
+    static {
+        DAY_NAME_MAP.put("monday", Day.MONDAY);
```

Comment:
I believe this should be in the `Day` class. Also lets discuss on a possible alternative implementation for this. But overall good idea!!